data/hai/hai outfits.txt

```diff
@@ -105,6 +130,64 @@ effect "ion impact"
 	"velocity scale" -.05
 
 
+outfit "Ionic Blaster Prototype"
+	category "Guns"
+	cost 530000
+	thumbnail "outfit/hai ionic blaster"
+	"mass" 28
+	"outfit space" -28
+	"weapon capacity" -28
+	"gun ports" -1
+	weapon
+		sprite "projectile/ion blast"
+			"frame rate" 5
+		sound "ion blast"
+		"hit effect" "ion impact" 2
+		"inaccuracy" 4
+		"velocity" 18
+		"lifetime" 40
+		"reload" 50
+		"firing energy" 170
+		"firing force" 20
+		"firing heat" 220
+		"hit force" 120
+		"shield damage" 150
+		"hull damage" 45
+		"ion damage" 1.5
+	description "Since the start of the war with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that goal, they began developing the Ionic Blaster, which does a small amount of ion damage that scrambles the weapons of the target, sometimes disabling it if its power generation is low enough. It also has enough raw damage to be useful in all situations."
+	description "The main focus they had in mind when developping the weapon was making sure they could fill Shield Beetles with them, unlike the bigger Ion Cannon. The Unfettered lack the resources to mass produce them, so it is only a prized possession of a lucky few."
```

Comment:
I'm not sure if this is the intended meaning, but I edited it so that the description would be clearer. Apologies if I changed the meaning.
```suggestion
	description "The main focus the Unfettered had in mind when developing the weapon was making sure they could fill all of a Shield Beetle's gun ports with them, unlike the bigger Ion Cannon. The Unfettered lack the resources to mass produce them, so it is only a prized possession of a lucky few."
```

Reply:
Well they can actually fill of a SB's gun ports with them, and it isnt far from the truth. Also you're right about it being clearer.